hw2/05-form.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>05 - Form</title>
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+    />
+    <script defer src="05-form.js"></script>
+  </head>
+  <body class="bg-dark">
+    <main>
+      <form class="bg-light border rounded w-50 mx-auto mt-5 p-3">
+        <h3 class="w-50 mx-auto mt-2 mb-4">Exercise 05 - Form</h3>
+        <div class="form-group">
+          <label class="fw-bold" for="name">Full Name:</label>
+          <input type="text" class="form-control mt-2" id="name" required />
```

Comment:
good idea on the required tag. I didn't think of that and I was checking the values inputted as I received them and then giving a consol error. I like your implementation more.

Reply:
Thanks! Yeah I thought it would be easier to do it this way